packages/react-native/index.ios.ts

```diff
@@ -1,14 +1,49 @@
-import { ReactNativeCommon } from './common';
+import { getCurrentBridge } from './bridge.ios';
+import { promisify, toNativeArguments } from './converter.ios';
+import RCTDeviceEventEmitter from './core/EventEmitter/RCTDeviceEventEmitter';
+import { getModuleMethods, isPromise, TModuleMethodsType, TNativeModuleMap } from './utils.ios';
+export { NativeEventEmitter } from './core/EventEmitter/NativeEventEmitter';
 
-export class ReactNative extends ReactNativeCommon {
-  bridge: RCTBridge;
-  init() {
-    if (!this.bridge) {
-      this.bridge = RCTBridge.alloc().init();
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const NativemModuleMap = require('@ammarahm-ed/react-native-podspecs/platforms/ios/lib/modulemap.json') as TNativeModuleMap;
+
+class NativeModuleHolder {
+  module: RCTBridgeModule;
+  moduleName: string;
+  _bridge: RCTBridge;
```

Comment:
If the `_` just means that this is intended to be private, we could declare it like this instead:
```suggestion
  private bridge: RCTBridge;
```